dts/msm8916/rules.mk

```diff
@@ -35,6 +35,8 @@ DTBS += \
 	$(LOCAL_DIR)/msm8916-samsung-r11.dtb \
 	$(LOCAL_DIR)/msm8929-samsung-r00.dtb \
 	$(LOCAL_DIR)/msm8929-samsung-r04.dtb \
+	$(LOCAL_DIR)/msm8239-samsung-r09.dtb \
```

Comment:
Please sort alphabetically. You should also squash two commits together (otherwise the first one doesn't compile) and give it a description in line with other such commits. Please also add the model to the list in the README :)